src/components/search/Search.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ChangeEvent, useState, useEffect } from "react";
+import styled from "styled-components";
+import SvgIcon, { IconType } from "../svg/SvgIcon";
+
+export interface SearchProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  id?: string;
+  name?: string;
+  onChangeCallback: (searchTerm: string) => void;
+  className?: string;
+  placeholder?: string;
+  debounceTime?: number;
+  width?: string;
+}
+
+const SearchContainer = styled.div<{ width: string }>`
+  position: relative;
+  display: inline-block;
+  width: ${(props) => props.width || "100%"};
+`;
+
+const SearchInput = styled.input`
+  outline: none;
+  padding-left: 30px;
+  width: 100%;
+  border-radius: 6px;
+  padding-top: 9px;
+  padding-bottom: 9px;
+  padding-right: 8px;
+  border: 1px solid var(--neutral-200, #9ca3af);
+  background: var(--system-50, #fff);
+  color: var(--neutral-200, #9ca3af);
+  box-shadow: 0px 1px 2px 0px rgba(0, 0, 0, 0.05);
+
+  &:focus {
+    border: 1px solid var(--Global-Tokens-Primary-100, #1c99ce);
+    outline: 1px solid var(--Global-Tokens-Primary-100, #1c99ce);
+  }
+
+  &:hover {
+    border: 1px solid var(--Global-Tokens-Primary-100, #1c99ce);
+  }
+`;
+
+const SearchIcon = styled(SvgIcon)`
```

Comment:
Can we name this to `StyledIcon`?
```suggestion
const StyledIcon = styled(SvgIcon)`
```